README.md

```diff
@@ -123,12 +138,15 @@ $ pyenv local <myvenv>
 Install the python packages described in the `requirements.txt`
 
 ```shell
-$ pip install -r requirements.txt
+$ cd requierements
+$ pip install -r developement.txt

```

Comment:
I think it is more appropriate to run this line instead.
```suggestion
$ pip install -r requirements/developement.txt
```
This prevents possible errors when moving from folder to folder.